tools/hardware/outputs/c_header.py

```diff
@@ -36,6 +36,15 @@
 #include <linker.h>         /* for BOOT_RODATA */
 #include <basic_types.h>    /* for p_region_t, kernel_frame_t (arch/types.h) */
 
+/* Wrap raw physBase location constant to give it a symbolic name in C that's
+ * visible to verification. This is necessary as there are no real constants
+ * in C except enums, and enums constants must fit in an int.
+ */
+static inline CONST word_t physBase(void)
```

Comment:
So, can't we use this?
```
const word_t physBase = PHYS_BASE_RAW;
```

Reply:
No, that will put a symbol in memory. The function will instead be compiled out like before.

Reply:
Ok, thanks for confirming.